### ISSUE TITLE
Install .tpk file on Samsung TV using a pendrive

### ISSUE THREAD

**Issue by jitendraAn** [flutter-tizen/flutter-tizen#562]
I want to install a .tpk file on Samsung TV using a pendrive, but when I open the pendrive on the Samsung TV, the only .tpk file doesn't show up. How can i install it? If anyone has any solution, please help me.

I am developing this App using Flutter for Tizen OS.

**Comment by swift-kim** [comment]
I don't have an answer to your question but I know a hacky way to update an _already installed_ application from USB if you managed to install it for the first time. That's not what you're looking for, right?

**Comment by jitendraAn** [comment]
Yes, but I want to know a hacky way to update an already-installed application from USB. How to do it.

**Comment by swift-kim** [comment]
Okay please email me.
